en/flight_modes/return.md

```diff
@@ -168,7 +168,7 @@ RC stick movement is ignored.
 ### VTOL
 
 VTOL aircraft use a [mission landing return type](#mission_landing_return) by default: 
-- If a mission landing is defined, fly direct to the mission landing start point and then land.
+- If a valid mission landing is defined, fly direct to the mission landing start point and then land.
```

Comment:
Is this true? VTOL uses mission landing by default? Why?

Reply:
Yes, because when you plan the VTOL mission appropriately (which is with a DO_LAND_START and a loiter to altitude before transition) then is he much safer/nicer landing compared to RTL_HOME.